eZ/Publish/Core/REST/Server/Input/Parser/Criterion/UserMetadata.php

```diff
@@ -30,6 +32,28 @@ class UserMetadata extends BaseParser
      */
     public function parse(array $data, ParsingDispatcher $parsingDispatcher)
     {
-        throw new \Exception('@todo implement');
+        if (!array_key_exists('UserMetadataCriterion', $data)) {
```

Comment:
We tend to prefer `!isset($data['UserMetadataCriterion'])` or `empty($data['UserMetadataCriterion'])` as long as null is not a valid value.